chaincode-labs/chaincode-podcast/privacy-and-robustness-in-lightning.md

```diff
@@ -331,112 +351,111 @@ When it's my turn, by definition, you haven't got anything in flight.
+Rusty Russell: 00:16:47
 
 Well, I think the any prep out cases have been pretty strong for a long time.
 The question's always been, do we want other soft fork as well?
 
 ## CTV
 
-Speaker 0: 00:16:54
+Rusty Russell: 00:16:54
 
-And people are like, oh, if we had obstacle the check sick from stack, we can also do it.
+And people are like, oh, if we had obstacle the `CHECKSIG` from stack, we can also do it.
 Sure.
 I can probably build a car out of matchsticks and snot, but I don't want to.
 And don't make me do it.
 
-Speaker 2: 00:17:09
+Adam Jonas: 00:17:09
 
 Could you do that?
 
-Speaker 0: 00:17:12
+Rusty Russell: 00:17:12
 
 A theoretical engineer probably could do that.
 I could not do that.
 I'm not crafty at all.
 But, okay.
 That is probably an aside.
 We don't need to go down.
-But, you know, check sync from stack is like, oh, it's cool that you can even make that work.
+But, you know, `CHECKSIG` from stack is like, it's cool that you can even make that work.
 But it is a terrible basis for just about anything.
 And it's more like a game we could play as far as what's the most obscure way we could do some kind of introspection.
-And the answer would be, you know, check-sig from stack.
-So any prep-out is a – and the proposals that are in that cluster, and there are a few, are pretty clean, pretty well understood.
-AJ would definitely be the person unfortunately he's not here in New York he's kind of you know done a fair bit of travel recently he was like no I'm not going.
+And the answer would be, `CHECKSIG` from stack.
+And the proposals that are in that cluster, and there are a few, are pretty clean, pretty well understood.
+AJ would definitely be the person unfortunately he's not here in New York he's kind of, done a fair bit of travel recently he was like no I'm not going.
 So I'm hoping that while we're in New York he is in his mountain lair somewhere coming up with a scheme to activate any prep out.
 That's if you're listening, AJ, I hope you're going to make that prophecy come true.
-I think, you know, we understand that we want this and we've wanted it for a fair period of time and nothing else has come up that's gone, oh no, we want this instead.
-Anyprepout by itself is a fairly simple piece.
+I think, we understand that we want this and we've wanted it for a fair period of time and nothing else has come up that's gone, oh no, we want this instead.
+Any prep out by itself is a fairly simple piece.
 
-Speaker 2: 00:18:21
+Adam Jonas: 00:18:21
 
-Now.
 Not a precluding piece to other things that we can do in the future.
 
-Speaker 0: 00:18:24
+Rusty Russell: 00:18:24
 
 Absolutely.
 
-Speaker 2: 00:18:25
+Adam Jonas: 00:18:25
 
 This is the two-way door, one-way door kind of thing.
 
-Speaker 0: 00:18:28
+Rusty Russell: 00:18:28
 
-In this case, we will want anyprepout even when we have full covenants and everything else.
+In this case, we will want any prep out even when we have full covenants and everything else.
```

Comment:
Same here, this refers to `ANYPREVOUT`, full name is `SIGHASH_ANYPREVOUT` but because speakers refer to it as `ANYPREVOUT`, it's okay to write just that.